docs/reference/remap/functions/format_int.cue

```diff
@@ -0,0 +1,45 @@
+package metadata
+
+remap: functions: format_int: {
+	category: "Number"
+	description: #"""
+		Formats the integer `value` into a string representation using the given base/radix.
+		"""#
+
+	arguments: [
+		{
+			name:        "value"
+			description: "The number to format as a string."
```

Comment:
As a string, but type = integer?

Reply:
True, `as` is ambiguous here 😄 I'll update it to just `The number to format`.